src/main/webapp/app/lecture/lecture-update.component.ts

```diff
@@ -82,6 +118,42 @@ export class LectureUpdateComponent implements OnInit {
                 this.isShowingWizardMode = params.shouldBeInWizardMode;
             }
         });
+
+        this.lectureOnInit = cloneDeep(this.lecture());
+    }
+
+    ngOnDestroy() {
+        this.subscriptions.unsubscribe();
+    }
+
+    isChangeMadeToTitleSection() {
+        return (
```

Comment:
Really a minor but for all these comparisons, you could consider using some functionality of `loadash` like `pick` and `isEqual`.

Reply:
I tried it, but this does not handle edge cases properly, so I think the current implementation is the easiest to understand 

When using this approach
```
    isChangeMadeToTitleSection() {
        const current = pick(this.lecture(), ['title', 'channelName', 'description']);
        const initial = pick(this.lectureOnInit, ['title', 'channelName', 'description']);
        return !isEqual(current, initial);
    }
```

The following test fails

```
        it('should handle undefined from description properly', () => {
            lectureUpdateComponent.lecture.set({ title: 'new title', channelName: 'new channel', description: 'new description' } as Lecture);
            lectureUpdateComponent.lectureOnInit = { title: 'old title', channelName: 'old channel', description: undefined } as Lecture;
            expect(lectureUpdateComponent.isChangeMadeToTitleSection()).toBeTrue();

            lectureUpdateComponent.lecture.set({
                title: lectureUpdateComponent.lectureOnInit.title,
                channelName: lectureUpdateComponent.lectureOnInit.channelName,
                description: '', // will be an empty string if the user clears the input, but was loaded with undefined in that case
            } as Lecture);
            expect(lectureUpdateComponent.isChangeMadeToTitleSection()).toBeFalse();
        });
```